javers-spring-boot-starter-redis/gradle.properties

```diff
@@ -0,0 +1,5 @@
+springVersion           =6.1.12
+springBootVersion       =3.3.3
```

Comment:
Normally, we use the same spring/spring-boot version in all modules, that version which is defined in the top level gradle.properties. If you need to bump it, it would be better to bump on the top level, for all modules.